app/_locales/en/messages.json

```diff
@@ -3590,6 +3590,9 @@
   "seedPhraseReq": {
     "message": "Secret Recovery Phrases contain 12, 15, 18, 21, or 24 words"
   },
+  "seedPhraseSettingsDescription": {
+    "message": "To keep your wallet safe, we'll ask a couple questions before revealing your Secret Recovery Phrase."
```

Comment:
I know it had a period in the comment but not sure if it was a mistake. I believe the guideline is to not use periods unless it's part of a multiple sentence description https://www.notion.so/MetaMask-Content-Design-Style-Guide-7d0b7579786540afbe02780ebb203254?pvs=4

<img width="748" alt="Screenshot 2023-06-05 at 3 25 24 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/333b7266-2ad7-453b-9999-940b8e41218d">

best check with @coreyjanssen